iceoryx_posh/CMakeLists.txt

```diff
@@ -235,9 +247,15 @@ endif(roudi_environment OR test)
 ########## exporting library ##########
 #
 setup_install_directories_and_export_package(
-    TARGETS iceoryx_posh iceoryx_posh_roudi RouDi
+    TARGETS iceoryx_posh iceoryx_posh_roudi
     INCLUDE_DIRECTORY include/
 )
+if(TOML_CONFIG)
+    setup_install_directories_and_export_package(
```

Comment:
I'm not sure this can be called twice, have to check

Reply:
Was working fine, when I tested it.